src/uwtools/tests/test_rocoto.py

```diff
@@ -244,12 +262,19 @@ def test__add_workflow_cycledef(self, instance, root):
             assert root[i].tag == "cycledef"
             assert root[i].text == item["spec"]
 
-    def test__add_workflow_log(self, instance, root):
-        path = "/path/to/logfile"
-        instance._add_workflow_log(e=root, logfile=path)
+    def test__add_workflow_log_basic(self, instance, root):
+        val = "/path/to/logfile"
+        instance._add_workflow_log(e=root, config={"log": val})
         log = root[0]
         assert log.tag == "log"
-        assert log.text == path
+        assert log.text == val
+
+    def test__add_workflow_log_cyclestr(self, instance, root):
+        val = "/path/to/logfile-@Y@m@d@H"
+        instance._add_workflow_log(e=root, config={"log": {"cyclestr": {"value": val}}})
+        log = root[0]
+        assert log.tag == "log"
+        assert log.xpath("cyclestr")[0].text == val
```

Comment:
Since `<log>` now supports `<cyclestr>`, test it both with a simple string value and with a `cyclestr` value.

Reply:
I clicked on `xpath` to learn a bit more about it and don't see any definitions/references to it currently.

Looking up a bit online, it looks like it's a "query language that can be used to search for and locate elements in XML and HTML documents. An XPath expression can be used to search through an XML document, and extract information from any part of the document" 

That's pretty cool usage here! It looks like it's imported with the `lxml` package.